test/sequential/test-module-loading.js

```diff
@@ -104,6 +104,15 @@ const d2 = require('../fixtures/b/d');
 assert.strictEqual(require('../fixtures/packages/index').ok, 'ok');
 assert.strictEqual(require('../fixtures/packages/main').ok, 'ok');
 assert.strictEqual(require('../fixtures/packages/main-index').ok, 'ok');
+assert.strictEqual(require('../fixtures/packages/missing-main').ok, 'ok');
+
+let unparseableErrorThrown = false;
+try {
+  require('../fixtures/packages/unparseable');
+} catch (e) {
+  unparseableErrorThrown = true;
+  assert.strictEqual(/^Error parsing .*/.test(e.message), true);
+}
```

Comment:
Lines 109 through 115 and line 330 should probably be removed and replaced with a call to `assert.throws()`:

```js
assert.throws(
  () => { require('../fixtures/packages/unparseable'); },
  /^Error parsing /
);
```

Reply:
Done.